app/stac_api/urls.py

```diff
@@ -1,18 +1,32 @@
+from django.conf import settings
 from django.urls import path
 
-from config.settings import API_BASE_PATH
 from stac_api.views import CollectionDetail
 from stac_api.views import CollectionList
+from stac_api.views import ItemDetail
+from stac_api.views import ItemsList
 
 from . import views
 
+API_BASE = settings.API_BASE_PATH
```

Comment:
`API_BASE` is called `API_BASE_PATH` below, maybe stick to one of the two and use the same in settings as well

Reply:
It was on purpose to make the code lighter, I'll rename it everywhere to `API_BASE` which is shorter ;-)

Reply:
but less descriptive :stuck_out_tongue: